### PR TITLE
OPM - fixed ORS issue

### PR DESCRIPTION
**Pull request by lwyszomi**
@czue @esoergel 

http://manage.dimagi.com/default.asp?187634#1051291

### CODE REVIEW

File: custom/opm/beneficiary.py [hunk 1]
```diff
@@ -489,6 +489,25 @@ def child_received_ors(self):
...
+            return True
+
+        for form in self.filtered_forms(CHILDREN_FORMS, months_before):
+            xpath = self.child_xpath('form/child_{num}/child{num}_child_orszntreat')
+            if form.get_data(xpath) == '0':
+                return False
+        return True
+
+
+    @property
```
Comment (dimagimon) [inline custom/opm/beneficiary.py]: E303 too many blank lines (2)

File: custom/opm/beneficiary.py [hunk 2]
```diff
@@ -489,6 +489,24 @@ def child_received_ors(self):
...
 
     @property
+    def child_received_ors_in_this_window(self):
+        months = self.child_age % 3
+        if months == 0:
+            months_before = 3
+        elif months == 1:
+            months_before = 1
+        else:
+            months_before = 2
```
Comment (czue) [inline custom/opm/beneficiary.py]: can you explain these numbers, maybe with a code comment? i don't follow the logic

**Comment by lwyszomi** [comment]
@czue Added comments. Is that make sense?

**Comment by esoergel** [comment]
Looks good to me.

**Comment by czue** [comment]
:+1: thanks!

**Pull request merged**